.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -17,6 +17,19 @@ body:
           required: true  
         - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
           required: true
+  - type: dropdown
+    attributes:
+      label: 'Issue Labels'
+      description: Please select a label that fits this bug report. Choose multiple, if applicable.
+      multiple: true
+      options:
+        - visual bug
+        - keyboard control not working
+        - text/string issue
+        - content not loading
+        - accessibility issue
```

Comment:
Maybe add another? What if the program gives you a bsod? What if it makes your motherboard melt? What option would you choose for those cases?

In all seriousness though, you should add "other issue" because not everything fits these categories.

Reply:
im just following instructions :'(
these are not all the labels these also need to be added

https://github.com/FreeTubeApp/FreeTube/pull/1593#issuecomment-921352760

Reply:
I'll mark this resolved but i didnt added it yet.

https://github.com/FreeTubeApp/FreeTube/pull/1593#issuecomment-930481244